stripe/src/main/java/com/stripe/android/view/CardInputView.java

```diff
@@ -46,13 +48,13 @@
     private ExpiryDateEditText mExpiryDateEditText;
     private LockableHorizontalScrollView mScrollView;
     private View mCardNumberSpace;
+    private boolean mCardNumberIsViewed;
```

Comment:
No real changes here - just alphabetization (and removing unnecessary initializer)